mp/src/game/shared/gamemovement.cpp

```diff
@@ -3931,26 +3931,34 @@ void CGameMovement::CheckFalling( void )
 	player->m_Local.m_flFallVelocity = 0.0f;
 }
 
-void CGameMovement::PlayerRoughLandingEffects( float fvol )
+void CGameMovement::PlayerRoughLandingEffects(float fvol, short surfacePropsIndex)
 {
+	surfacedata_t* surfaceData;
+
 	if ( fvol > 0.0 )
```

Comment:
No need to declare the pointer outside of this if block, it only gets used inside of it, and use `pSurfaceData` instead.

Reply:
I tried my best to match the convention of the other functions in the file, should I use the momentum guidelines when editing files that came with the SDK?

Reply:
Yes. We're not Valve so we can start to do our own bits, even in SDK code.